### PR TITLE
grammalecte: 0.5.17.2 -> 0.5.18

### PR DESCRIPTION
**Pull request by apeyroux**
###### Motivation for this change

grammalecte: 0.5.17.2 -> 0.5.18

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/grammalecte/default.nix [hunk 1]
```diff
@@ -31,7 +30,7 @@ buildPythonPackage rec {
 
   meta = {
     description = "Grammalecte is an open source grammar checker for the French language";
-    homepage = https://dicollecte.org/grammalecte/;
+    homepage = "https://dicollecte.org/grammalecte/";
```
Comment (joachifm) [inline pkgs/development/python-modules/grammalecte/default.nix]: Note that quoting urls like this is unnecessary

**Comment by joachifm** [comment]
Can you squash the commits? I could do it for you but that'd also remove your signature (if that matters to you).

**Comment by apeyroux** [comment]
You can do it, I do not mind that.

**Comment by joachifm** [comment]
Merged. Thank you

**Pull request merged**